lib/config.js

```diff
@@ -40,7 +43,7 @@ const getConfig = async (args) => {
       const { managementToken, activeSpaceId, activeEnvironmentId, host } = config || {};
       contentfulCliOptions = {
         spaceId: activeSpaceId,
-        accessToken: managementToken,
+        managementToken: managementToken,
```

Comment:
Probably also should be documented, that we now expect managementToken as the name (but still support accessToken?)